config/routes.rb

```diff
@@ -5,6 +5,7 @@
   resources :adoption_applications, only: [:new, :create]
   resources :volunteer_applications, only: [:new, :create]
   resources :newsletters, only: [:index]
+  resources :contact_messages, only: [:new, :create]
```

Comment:
I also try and keep the top level `resources` alphabetized and grouped, so I would have an alphabetized list of `resources` then another alphabetized list of single `resource` routes.